nodego/events.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodego
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"google.golang.org/api/pubsub/v1"
```

Comment:
Especially since we don't suggest installing with go install, it might be nice to allow use of the HTTP version without getting these imports. Maybe this event code could go in a separate package which only needed to be imported if using pubsub/GCS?

Reply:
Would this be a big problem? I believe `go build` will only include code in the binary if they are used.

Reply:
Imports can have side effects via inits, so this will fail to build without the imported code being available.